R/Rules-methods.R

```diff
@@ -2616,7 +2616,7 @@ setMethod("stopTrial",
       ## Find the variance of the log of the dose_target_samples(eta)
       M1 <- matrix(c(-1 / (model@phi2), -(log(prob_target / (1 - prob_target)) - model@phi1) / (model@phi2)^2), 1, 2)
       M2 <- model@Pcov
-      varEta <- M1 %*% M2 %*% t(M1)
+      varEta <- as.vector(M1 %*% M2 %*% t(M1))
 
       ## Find the upper and lower limit of the 95% credibility interval
       CI <- exp(log(dose_target_samples) + c(-1.96, 1.96) * sqrt(varEta))
```

Comment:
may consider
```suggestion
      CI <- exp(log(dose_target_samples) + c(-1, 1) * 1.96 * sqrt(varEta))
```
here and below